src/data/content/homepage.ts

```diff
@@ -9,78 +9,46 @@ export default {
     image: 'https://files.ably.io/website/images/meta-tags/ably-generic%402x.jpg',
     twitter: '@ablyrealtime',
   },
-  sections: [
-    {
-      columns: 1,
-      bottomMargin: 48,
-      cards: [
-        {
-          title: 'Ably Documentation',
-          type: 'hero',
-          content: 'Realtime experience infrastructure that just works at any scale.',
-        },
-      ],
+  platformProducts: {
+    title: 'Platform & Products',
+    platformCard: {
+      title: 'Platform',
+      content: "Understand the core concepts of Ably's platform.",
+      image: 'platform.png',
+      link: '/docs/platform',
     },
-    {
-      columns: 2,
-      bottomMargin: 80,
-      cards: [
-        {
-          title: 'Platform',
-          type: 'feature',
-          content: "Understand the core concepts of Ably's platform, its pricing and integrations.",
-          image: 'platform.png',
-          links: [{ text: 'Get started', href: '/docs/platform' }],
-        },
-        {
-          title: 'Pub/Sub',
-          type: 'feature',
-          content: "Quickly create realtime digital experiences with Ably's core building blocks.",
-          image: 'pub-sub.png',
-          links: [{ text: 'Get started', href: '/docs/basics' }],
-        },
-        {
-          title: 'Chat',
-          type: 'feature',
-          content: 'Quickly add chat features into any application.',
-          image: 'chat.png',
-          links: [{ text: 'Get started', href: '/docs/chat' }],
-        },
-        {
-          title: 'Spaces',
-          type: 'feature',
-          content: 'Build collaborative environments in just a few lines of code.',
-          image: 'spaces.png',
-          links: [{ text: 'Get started', href: '/docs/spaces' }],
-        },
-        {
-          title: 'LiveSync',
-          type: 'feature',
-          content: 'Seamlessly sync database changes with frontend clients.',
-          image: 'livesync.png',
-          links: [{ text: 'Get started', href: '/docs/livesync' }],
-        },
-        {
-          title: 'Asset Tracking',
-          type: 'feature',
-          content: 'Track the location of assets in realtime.',
-          image: 'asset-tracking.png',
-          links: [{ text: 'Get started', href: '/docs/asset-tracking' }],
-        },
-      ],
-    },
-    {
-      columns: 1,
-      bottomMargin: 160,
-      cards: [
-        {
-          title: 'SDKs',
-          content: 'Ably SDKs provide a consistent and idiomatic API across a variety of supported platforms.',
-          image: 'sdks@2x.png',
-          type: 'sdk',
-          callToAction: { text: 'View all SDKs', href: '/docs/sdks' },
-        },
-      ],
-    },
-  ],
+    productCards: [
+      {
+        name: 'pubsub',
+        link: '/docs/basics',
```

Comment:
just thinking is this the updated link? why cant we just update the link in ably-ui so we can use the products  from ably-ui ?

Reply:
These cards do different things in different places - they may not always be links. On `/pubsub` for instance, they control the Composable Realtime component and don't link to anything (a CTA inside does instead). Therefore, we shouldn't make this behaviour default on the ProductTile side

Reply:
@aralovelace we maybe want the docs product links to point to docs and the meganav to point to the product pages

Reply:
meganav goes to /chat and docs will link to /docs/chat

Reply:
@denissellu  yes I think the default link are linked to docs

https://github.com/ably/ably-ui/blob/c74dac919036113f5385c8d373c9f8aeb81f5efe/src/core/ProductTile/data.ts#L26

Reply:
ah i see, we can keep this option here in case in the future they would like to change it